### ISSUE TITLE
Conversations are backuped in INTERNAL memory even if SD card is installed

### ISSUE THREAD

**Issue by klint** [signalapp/Signal-Android#10696]
- [X ] I have searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [X ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Conversation backup does not ask for backup location when SD card installed.
Backups are saved on internal card.

### Steps to reproduce
- Make sure an SD Card is installed in the phone
- Open Signal 
- Go to Parameters/Conversations (sorry, Signal is in French for me, I don't know the exact wording in the English version)
- Click on "Backup conversations"
- If backups are already active, then deactivate them
- Click on the Activate button

**Actual result:** A pop up is displayed showing the passphrase. Then, when proceeding with the backup, it appears that the target folder is signal/backups on the INTERNAL card. 
**Expected result:** A pop up should be displayed to ask for the backup target folder which can then be chosen on the SD Card.

### Device info

**Device:** Redmi Note 7 (with SD Card installed)
**Android version:** 9
**Signal version:** 5.2.2

### Link to debug log

https://debuglogs.org/83031037fd01be7c1650630c5554d0355b9ebfaabfa1d0b4e1a428b2262a4786

**Comment by Leptopoda** [comment]
Hi there and welcome
this is not a bug rather a missing feature of sorts
Feature requests are made and discussed in the [community forum](https://community.signalusers.org) and there is already a thread about that here
https://community.signalusers.org/t/allow-backups-on-the-sd-card/3524
There might even bee other requests just join us there to start discussing.

Some further background:
Signal made a change allowing backups to custom folders as you mentioned around November 2020. 
The problem was that many SD-cards have problems with files larger than 4GB (especially FAT filesystems). As many have backups larger than the mentioned size (mine is around  6GB rn) the backups would fail.
Theoretically this should be a thing of the past fith ExFAT suooprting 'ulimited' file sizes but mybe there is some other constraint still in android.
Please join the discussion(s) in the forum and close this issue because it isn't really a bug.

**Comment by klint** [comment]
Hi Leptopoda. Thanks for the clarification.

Then there is another bug, which is that the Signal app clearly displays the ability to save backups on an external memory, even if it cannot because of the filesystem type : "Sauvegarde des conversations / Sauvergarder les conversations dans la mémoire externe". The message should be more specific than it is now (at least in French), indicating which file format are supported (with a link to a support page?). Shall I open another bug for that here? Or just rename this one? 

Also, it it correct to say that if I reformat my SD Card to ExFat, then the backups will be able to be stored on the SD card? (I known another phone where the option is really available, but I have not my hand on it now to check to filesystem there).

Thanks :)

**Comment by klint** [comment]
So it is a bug actually: the app is not able to something that is clearly advertised.
I have checked my own SD card, and it is ExFAT. I'll have to check the other phone's one.
Actually, with 40 millions of additional users, and especially in comparison to Whatsapp, backup will be a real problem. That is something that new users will notice once it is too late (when they have lost all their conversations (and medias) in a crash of their phones, without any backup ;) ). I don't want to make Signal quite similar to Whatsapp, but backup is a key feature. :)
Closing for now (and opening a new bug for the misleading wording).
